cmd/ctr/pprof.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"errors"
```

Comment:
I just don't  think the original code : "%s" make any sense. 
either ` fmt.Errorf(resp.Status)` or `errors.New(resp.Status)` could be more direct and simpler